apps/soperator/data.yaml

```
@@ -0,0 +1,240 @@
+tags:
+  - AI/Machine Learning
+title: Nebius Soperator - Kubernetes Operator for SLURM
+summary: "Enjoy the best out of two worlds for massive HPC workloads"
+logo: ./assets/slurm_logo.svg 
+created: "2025-11-3T10:00:00Z"
+description: |
+    Both Slurm and Kubernetes can serve as workload managers for distributed model training and high-performance computing (HPC) in general.
+    Each of these systems has its strengths and weaknesses, and the trade-offs between them are significant. 
+    Slurm offers advanced and effective scheduling, granular hardware control, and accounting, but lacks universality. 
+    On the other hand, Kubernetes can be used for purposes other than training (e.g. inference) and provides good auto-scaling and self-healing capabilities.
+
+
+    The end-to-end deployment requires two Helm charts in addition to the prerequisites:
+
+    1. Slurm Operator (soperator)
+    2. Slurm Cluster 
+support_link: https://github.com/nebius/soperator
+support_type: Enterprise
+charts:
+  - name: helm-soperator
+    versions: ['1.21.1']
+  - name: helm-slurm-cluster
+    versions: ['1.21.1']
+
+prerequisites: |
+  **Soperator requires**:
+
+  * [K0rdent - 1.4.0](https://docs.k0rdent.io/v1.4.0/admin/installation/install-k0rdent/) 
```

```suggestion
  * [k0rdent >= 1.4.0](https://docs.k0rdent.io/latest/admin/installation/install-k0rdent/) 
```
- AFAIK k0rdent should always start with lower "k".
- I guess it should be supported in higher k0rdent versions too, right? So we should adjust k0rdent version accordingly.